applications/crossbar/src/modules/cb_alerts.erl

```diff
@@ -267,14 +298,14 @@ do_check_port_requests([PortRequest|PortRequests], Context) ->
                           ,Routines
                           ),
     do_check_port_requests(PortRequests, Context1);
-do_check_port_requests(OwnerId, Context) ->
-    case knm_port_request:account_active_ports(OwnerId) of
-        {'error', _R} ->
-            lager:debug("unable to fetch port requests: ~p", [_R]),
-            Context;
-        {'ok', PortRequests} ->
-            do_check_port_requests(PortRequests, Context)
-    end.
+do_check_port_requests({'ok', PortRequests}, Context) ->
+    do_check_port_requests(PortRequests, Context);
+do_check_port_requests({'error', _R}, Context) ->
+    lager:debug("unable to fetch port requests: ~p", [_R]),
+    Context;
+do_check_port_requests(Unexpected, Context) ->
+    lager:debug("unable to fetch port requests: ~p", [Unexpected]),
+    Context.
```

Comment:
Remove these clauses. This function is always getting a list of jobjs.

Reply:
okay it seems you're calling it from test with `{ok,....` but it doesn't make sense because in the tst you can call this with Actives and be done with it. Also widening function clauses for such a thing that maybe is called from test it bad!